lib/services/ios-debug-service.ts

```diff
@@ -72,14 +74,15 @@ class IOSDebugService implements IDebugService {
         }).future<void>()();
     }
 
-    private emulatorDebugBrk(): IFuture<void> {
+    private emulatorDebugBrk(shouldBreak? : boolean): IFuture<void> {
```

Comment:
when we have to pass boolean options to method, we prefer using objects and pass a property of them:
```TypeScript
 private emulatorDebugBrk(debugOptions?: {shouldBreak : boolean}): IFuture<void> {
```
So the usage will be:
```TypeScript
this.emulatorDebugBrk().wait()
this.emulatorDebugBrk({shouldBreak: true}).wait();
```
This way it's pretty easy to understand the purpose of the passed value. (passing only `true` does not give you this information)

And in the method you'll make the check:
```TypeScript
let args = debugOptions && debugOptions.shouldBreak ? "--nativescript-debug-brk" : "--nativescript-debug-start";
```